content/client-server-api/_index.md

```diff
@@ -13,20 +13,33 @@ clients which maintain a full local persistent copy of server state.
 ## API Standards
 
 The mandatory baseline for client-server communication in Matrix is
-exchanging JSON objects over HTTP APIs. HTTPS is recommended for
-communication, although HTTP may be supported as a fallback to support
-basic HTTP clients. More efficient optional transports will in future be
-supported as optional extensions - e.g. a packed binary encoding over
-stream-cipher encrypted TCP socket for low-bandwidth/low-roundtrip
-mobile usage. For the default HTTP transport, all API calls use a
-Content-Type of `application/json`. In addition, all strings MUST be
-encoded as UTF-8.
+exchanging JSON objects over HTTP APIs. More efficient transports may be
+specified in future as optional extensions.
+
+HTTPS is recommended for communication. The use of plain HTTP is not
+recommended outside test environments.
 
 Clients are authenticated using opaque `access_token` strings (see [Client
 Authentication](#client-authentication) for details).
 
+All `POST` and `PUT` endpoints, with the exception of
+[`POST /_matrix/media/v3/upload`](#post_matrixmediav3upload), require the
+client to supply a request body containing a (potentially empty) JSON object.
+Clients should supply a `Content-Type` header of `application/json` for all requests with JSON bodies,
+but this is not required.
+
+Similarly, all endpoints require the server to return a JSON object,
+with the exception of 200 responses to
+[`GET /_matrix/media/v3/download/{serverName}/{mediaId}`](#get_matrixmediav3downloadservernamemediaid)
+and [`GET /_matrix/media/v3/thumbnail/{serverName}/{mediaId}`](#get_matrixmediav3thumbnailservernamemediaid).
+Servers msut include a `Content-Type` header of `application/json` for all JSON responses.
```

Comment:
s/msut/must/

Reply:
fixed in https://github.com/matrix-org/matrix-spec/commit/3dc3150f6d8c1bf77bd41af2b7540c1632d18998